_pages/en/target-solution-delivery-model-strategy-areas-of-concerns.md

```diff
@@ -0,0 +1,416 @@
+  - [2.7. Alignment and Gap analysis with Policy and Directive (Investment & Project Management)](#27-alignment-and-gap-analysis-with-policy-and-directive-investment--project-management)
+  - [2.8. Sustainability of skill sets](#28-sustainability-of-skill-sets)
+  - [2.9. Enterprise Resource Planning (ERP) solutions](#29-enterprise-resource-planning-erp-solutions)
+  - [2.10. Risk to project management overhead](#210-risk-to-project-management-overhead)
+  - [2.11. Risk to capability-based planning ability](#211-risk-to-capability-based-planning-ability)
+  - [2.12. Risk to enterprise reference data model and standard compliance](#212-risk-to-enterprise-reference-data-model-and-standard-compliance)
+- [Appendix A - Archived](#appendix-a---archived)
+
+## 1. Document Purpose
+
+To document the areas of concerns between stakeholders involved in the Alternate IT Solution Delivery Model Strategy.
+
+Why?
+
+With that many stakeholders involved in the elaboration and affected by the strategy, two realities need to be managed:
+
+1. Important feedbacks risk being lost during peer reviews, and
+2. Consensus is unlikely.
+
+During the strategy's peer review process, important feedbacks that are unable to be addressed at that time need to be tracked so the continuing elaboration of the strategy do not forget them.
+A strategy is words on a page.
+If stakeholders do not buy in to it, things won't change.
+
+Consensus is unlikely as choices will have to be made in order to keep moving forward.
+Those choices may contradict a stakeholder's position.
+To maintain transparency amongst stakeholders and to the authority of this strategy (expected to be the CIO), a list of those choices will be kept with sufficient justifications to explain them.
+
+## 2. Areas of concerns
+
+### 2.1. IT Solutions that spans more than one project
+
+#### 2.1.1. Summary <!-- omit in toc -->
+
+One IT solution may span more than one project and include work involved to satisfy multiple project objectives.
+For example, data management is a capability that crosses multiple project initiative and is required as part of the architecture, design, and implementation of an IT solutions.
+
+How to manage and keep track of such cross-project work?
+
+#### 2.1.2. Raised by stakeholder(s) <!-- omit in toc -->
+
+- Enterprise Architecture
+- Chief Data Office
+
+#### 2.1.3. What is being done about this concern <!-- omit in toc -->
+
+*Coordinating activities*
+
+If a given project requires a given competency, then that competency should exist within the team.
+As a result, if data science is a competency that is required for a given value stream to deliver value to its users, then data science capabilities should be embedded within the team.
+As for how those services are delivered, the technologies should be in place through internal such that the team is able to manage the software required to deliver the data analysis capabilities they need.
```

Comment:
Worked on the format and overall typos. I think I got most of them but I'm not 100% what you were referring to here:

> the technologies should be in place through **internal** such that

Reply:
oh oh @jaysonmc ? :)